packages/jsii-pacmak/lib/targets/dotnet/dotnetgenerator.ts

```diff
@@ -294,7 +293,7 @@ export class DotNetGenerator extends Generator {
     this.dotnetRuntimeGenerator.emitAttributesForClass(cls);
 
     this.code.openBlock(
-      `public${inner}${absPrefix} class ${className}${implementsExpr}`,
+      `public${absPrefix} class ${className}${implementsExpr}`,
```

Comment:
C# does not allow static member classes to declare protected members (our contract includes some, so that was bad). The presence of the `static` keyword here was a Java-ism, since nested classes in C# (with or without `static`) are actually semantically related to Java static nested classes (the syntactic sugar that Java has for non-static nested classes does not exist in C#).

Reply:
This should have been a separate PR, don'tcha agree?

Reply:
Except that separate PR's test would not have been able to pass without the other changes in this PR, and vice-versa...